### PR TITLE
Epo playbooks new arch

### PR DESCRIPTION
**Pull request by gshriki**
ePO playbooks looks better, and using the new ePO and ServiceNow integrations

### CODE REVIEW

**Comment by aviadl** [comment]
Do not merge till ServiceNow is merged

**Comment by JonathanMeler** [comment]
currently 
image

set all `is system task` set to `false`

**Comment by aviadl** [comment]
@JonathanMeler This is fine, load it via content update it will work properly

File: Playbooks/playbook-McAfee_ePO_Endpoint_Compliance_Playbook.yml [hunk 1]
```diff
@@ -246,21 +300,138 @@ tasks:
...
+    task:
+      id: 370580ad-016e-4515-8a0e-e76cd0e55373
+      version: -1
+      name: Store system names to context key outdatedEndpoints
+      description: Takes the comments of a given entry ID and stores them in the
+        incident context, under a provided context key.
+        For accessing the last executed
+        task's comments, provide ${lastCompletedTaskEntries.[0]} as the value for
+        the entryId input parameter.
+      scriptName: commentsToContextV2
```
Comment (JonathanMeler) [inline Playbooks/playbook-McAfee_ePO_Endpoint_Compliance_Playbook.yml]: What is this script? it's not pushed to demo2. (call from all playbooks)

**Pull request merged**